stream-chat-android-offline/src/test/java/io/getstream/chat/android/offline/Mother.kt

```diff
@@ -499,7 +499,7 @@ internal fun randomReaction(
 )
 
 internal fun randomSyncStatus(exclude: List<SyncStatus> = emptyList()): SyncStatus =
-    (SyncStatus.values().asList() - exclude).random()
+    (SyncStatus.values().asList() - exclude - SyncStatus.WAIT_ATTACHMENTS).random()
```

Comment:
exclude this new value since it's state related to message only with attachments